docs/Learning-Environment-Design-Agents.md

```diff
@@ -430,7 +430,7 @@ be unique for that Agent, but multiple Agents can have a Sensor Component with
 the same name).
 
 Visual observations also support stacking, by specifying `Observation Stacks`
-to a value greater than 1. The visual observations from the last `stackSize-1`
+to a value greater than 1. The visual observations from the last `stackSize`
```

Comment:
Maybe "The visual observations from current step and the last`stackSize-1` steps"? This is OK too though.